### ISSUE TITLE
Move projects to their proper place, add READMEs

### ISSUE THREAD

**Issue by shcheklein** [iterative/dataset-registry#34]
- DVC repos into `docs/dvc` ... cc @jorgeorpinel  (low priority)
- [github-issues](https://github.com/iterative/dataset-registry/tree/master/github-issues) cc @ykasimov @mnrozhkov do we still need it? Do you plan to do a blog post?
- [fashion-mnist](https://github.com/iterative/dataset-registry/tree/master/fashion-mnist) - remove if /when not used cc @jorgeorpinel @alex000kim @dberenbaum  - I think it was part of the Get Started? May be not used anymore.
- mnist - the same - do we need it, where?

**Comment by dberenbaum** [comment]
I don't think that fashion-mnist is being used anywhere

**Comment by dberenbaum** [comment]
> * DVC repos into `docs/dvc` ... cc @jorgeorpinel  (low priority)

I looked into it as part of #35, but when I started looking through the docs, I was concerned that going from `get-started/data.xml` -> `docs/dvc/get-started/data.xml` or similar everywhere makes all the paths look too long.

**Comment by shcheklein** [comment]
Yes, @dberenbaum it makes sense. Yes, let's keep it simpler, removed the item.

**Comment by jorgeorpinel** [comment]
See https://github.com/iterative/dataset-registry/pull/37.
